opentelemetry-auto-instrumentation/test/opentelemetry-auto-instrumentation_test.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe 'AutoInstrumentation' do
+  before do
+    OpenTelemetry::TestHelpers.reset_opentelemetry
+  end
+
+  after do
+    OTelBundlerPatch.send(:remove_const, :OTEL_INSTRUMENTATION_MAP)
```

Comment:
A side effect of localizing the methods. will mean only having to remove a single method from Bundler patch.